docset/winserver2022-ps/pki/New-SelfSignedCertificate.md

```diff
@@ -810,147 +965,153 @@ Accept wildcard characters: False
 ```
 
 ### -TextExtension
-Specifies an array of certificate extensions, as strings, which this cmdlet includes in the new certificate.
-Each string must employ one of the following formats: 
 
-oid`=`base64String, where oid is the object identifier of the extension and base64String is a value that you provide.
-After decoding base64String, the value must be valid Abstract Syntax Notation One (ASN.1).
-For more information, see [Abstract Syntax Notation One (ASN.1): Specification of basic notation](http://www.itu.int/ITU-T/studygroups/com17/languages/X.680-0207.pdf).
+Specifies an array of certificate extensions, as strings, which this cmdlet includes in the new
+certificate. Each string must employ one of the following formats:
+
+oid`=`base64String, where oid is the object identifier of the extension and base64String is a value
```

Comment:
@sdwheeler I'm unsure how `oid` and `base64String` placeholder text should be handled in this case:

```markdown
oid`=`base64String, where oid is the object identifier of the extension and base64String is a value
```

I considered treating them like PowerShell variables. Any thoughts?

```suggestion
`"$oid=$base64String"`, where `$oid` is the object identifier of the extension and `$base64String` is a value
```

There are similar examples in the help for this parameter on lines 977 and 981 and a reference to `oid=` on line 986 in this PR.

Reply:
Here is my suggested formatting for this case:

```
- `{oid}={base64String}`, where **oid** is the object identifier of the extension and
  **base64String** is a value that you provide. After decoding base64String, the value must be valid
  Abstract Syntax Notation One (ASN.1). For more information, see
  [Abstract Syntax Notation One (ASN.1): Specification of basic notation](http://www.itu.int/ITU-T/studygroups/com17/languages/X.680-0207.pdf).

- `{oid}={hex}`, where **oid** is the object identifier of the extension and **hex** is a
  hexidecimalString value that you provide. After decoding hexidecimalString, the value must be
  valid ASN.1.

- `{oid}={text}`, where **oid** is the object identifier of the extension and **text** is a string
  value that you provide. The string must contain a textual representation of the extension value in
  a format specific to each object ID. When the string is processed, it will be encoded into an
  ASN.1 extension value before being placed into the new certificate as an extension.
```